pom.xml

```diff
@@ -13,6 +13,11 @@
   <name>Jenkins jQuery plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.625</jenkins.version>
+    <java.level>6</java.level>
```

Comment:
As far as I am concerned, if you set 1.625.3 here, you don't have the possibility to install the plugin on a 1.625 instance. Greater versions should be compatible with minor ones.

With this property you are setting  the minimum required version, and if you are bumping it, better require one that has less error than the weekly one.

Probably @andresrc can give us some clues here.